payloads/v8/interactions.ts

```diff
@@ -368,6 +395,15 @@ export interface APIInteractionResponseDeferredChannelMessageWithSource {
 	data?: Pick<APIInteractionApplicationCommandCallbackData, 'flags'>;
 }
 
+export interface APIInteractionResponseDeferredMessageUpdate {
+	type: InteractionResponseType.DeferredMessageUpdate;
+}
+
+export interface APIInteractionResponseUpdateMessage {
+	type: InteractionResponseType.UpdateMessage;
+	data?: APIInteractionApplicationCommandCallbackData;
```

Comment:
Just so we're sure, is data *optional*?

Reply:
It is I just checked, you can send a type 7 with no data and the interaction will just complete with no error. The message will technically be "edited" in that its update time changes and it will gain an (edited) if there isn't one, but it doesn't change.